src/cgame/cg_rocket_draw.cpp

```diff
@@ -1809,7 +1809,7 @@ class CenterPrintElement : public HudElement
 			showTime_ = cg.centerPrintTime;
 			SetInnerRML( Rocket_QuakeToRML( cg.centerPrint, RP_EMOTICONS ) );
 			SetProperty( Rml::Core::PropertyId::FontSize,
-			             Rml::Core::Property( 0.2f * cg.centerPrintSize, Rml::Core::Property::EM ) );
+			             Rml::Core::Property( 0.07f * cg.centerPrintSize, Rml::Core::Property::EM ) );
```

Comment:
This makes the smallest center prints unreadably small. For an example of the smallest ones (size 10) you can do `/delay 1000 buy lgun` as a human while not having enough funds. To see one of the largest ones (size 30), destroy all your spawns.

So what needs to happen is changing some of the sizes in calls to `CG_CenterPrint` to decrease the size difference between the smallest and largest ones.